sc/shared/src/test/scala/sigmastate/ErgoTreeSpecification.scala

```diff
@@ -367,7 +367,7 @@ class ErgoTreeSpecification extends SigmaDslTesting with ContractsTestkit {
         MInfo(4, BytesWithoutRefMethod),
         MInfo(5, IdMethod),
         MInfo(6, creationInfoMethod),
-        MInfo(7, getRegMethod),
+        MInfo(7, getRegMethodV5),
```

Comment:
It is better to version this test as well.

Reply:
done